memote/support/helpers.py

```diff
@@ -173,8 +208,32 @@ def find_biomass_reaction(model):
     model : cobra.Model
         The metabolic model under investigation.
 
+    Returns
+    -------
+    list of biomass reactions
+
     """
-    return [rxn for rxn in model.reactions if "biomass" in rxn.id.lower()]
+    buzzwords = ['biomass', 'growth', 'bof']
+
+    buzzword_matches = set([rxn for rxn in model.reactions if any(
+        string in rxn.id.lower() for string in buzzwords
+    )])
+
+    sbo_matches = set([rxn for rxn in model.reactions if
+                       rxn.annotation is not None and
+                       'SBO' in rxn.annotation and
+                       rxn.annotation['SBO'] == 'SBO:0000630'])
+
+    regex = re.compile('^{}(_[a-zA-Z]+?)*?$'.format('biomass'), re.IGNORECASE)
```

Comment:
I would use the same method as you did above for `id` and `name` or query with a simpler regex. Regex's are expensive, though, so overall maybe:
```python
for element in model.reactions|metabolites:
    for word in buzzwords:
        if word in element.id.lower() or word in element.name.lower():
            hits.append(element)
```